lib/lexorank/rankable.rb

```diff
@@ -60,9 +60,11 @@ def move_to(position)
         if position == 0
           [nil, collection.first]
         else
+          current_position = collection.map(&:id).index(self.id)
```

Comment:
Not in scope of this PR but I didn't like this solution in first place because it's operating on the collection within ruby which is very costly in terms of performance. 

However, previously it was only called if needed. Now it is executed every time if the position is not 0. For example running the scope benchmark, that creates 5000 records, will run around ~15x slower than before (at least on my machine).

Maybe we can change it to only run if `no_rank?` is false. This would preserve the original behavior. I still need to figure out a way around this thing entirely...

Reply:
Well I just thought about it for 5 minutes and I think I have a solution:

The only case where we want to use `collection.offset(position).limit(2)` is when the item has a rank and is in front of the collection. We can just avoid all the checks and use 

```ruby
collection.where.not(id: self.id).offset(position - 1).limit(2)
```

The tests tell me that this seems to work. I'm going to merge your PR, make the aforementioned change and am going to release a new version of the gem!